worlds/AutoWorld.py

```diff
@@ -115,6 +119,10 @@ def call_single(multiworld: "MultiWorld", method_name: str, player: int, *args:
             logging.error(message)
         raise e
     else:
+        taken = time.perf_counter()-start
+        if taken > 0.1:
```

Comment:
I think these might be confusing to users so should probably not run or at least be logged when optimized.
```suggestion
        if taken > 0.1 and __debug__:
```

Reply:
I disagree